test/support/commands.js

```diff
@@ -4,11 +4,17 @@ Cypress.Commands.add('getPreviewPage', (pageNum) => {
     cy
         .get(`.page[data-page-number=${pageNum}]`)
         .as('previewPage')
-        .find('[data-testid="page-loading-indicator"]')
+        // Adding 10s timeout here because there's no reliable way to wait for
```

Comment:
Using the loading indicator seems reliable, but the document just might take longer to load/render than the default timeout allows.